### ISSUE TITLE
Received `true` for a non-boolean attribute `infinite`.

### ISSUE THREAD

**Issue by ErikParso** [express-labs/pure-react-carousel#212]
Got following error:

**Warning: Received `false` for a non-boolean attribute `infinite`.**
If you want to write it to the DOM, pass a string instead: infinite="false" or infinite={value.toString()}.
If you used to conditionally omit it with infinite={condition && value}, pass infinite={condition ? value : undefined} instead.
    in div (created by t)
    in t (created by i)
    in i (created by Gallery)
    in div (created by t)
    in t (created by Gallery)
    in div (created by Gallery)
    in Gallery (created by CarDetailPage)
    ...

I am using react with typescript. My carousel looks like this:

```
<CarouselProvider
    className={styles.carousel}
    naturalSlideWidth={100}
    naturalSlideHeight={125}
    totalSlides={this.props.images.length}>
    <Slider>
        {
            this.props.images.map((src, i) =>
                <Slide key={i} index={i}><img src={src} /></Slide>)
        }
    </Slider>
</CarouselProvider>
```

**Comment by mrbinky3000** [comment]
I think this may be fixed in a recent release.  There was a spot where we forgot to filter out the "infinite" prop before passing it to a DOM element.

**Comment by ErikParso** [comment]
great, thank you.

**Comment by pavelusov** [comment]
We are waiting!

**Comment by bcarroll22** [comment]
This is fixed in ^1.24.0
